docs/assets/js/main.js

```diff
@@ -0,0 +1,14 @@
+let acc = document.getElementsByClassName("video_accordion");
+var i;
+
+for (i = 0; i < acc.length; i++) {
+    acc[i].addEventListener("click", function () {
+        this.classList.toggle("active");
+        var panel = this.nextElementSibling;
```

Comment:
since the value of panel does not change, a const will be preferred in this case. const panel - this.nextElementSibling;